ProcessLib/ComponentTransport/CreateComponentTransportProcess.cpp

```diff
@@ -53,28 +54,45 @@ std::unique_ptr<Process> createComponentTransportProcess(
 
     std::vector<std::vector<std::reference_wrapper<ProcessVariable>>>
         process_variables;
+
+    // get all process variables stored in a vector before allocation
+    // pressure first, concentration then
+    auto const unalloc_process_variables = findProcessVariables(
```

Comment:
What do you mean by `unalloc`?

Reply:
Given a more accurate name for this variable.